CMakeLists.txt

```diff
@@ -0,0 +1,34 @@
+﻿cmake_minimum_required (VERSION 3.4)
+set(CMAKE_OSX_DEPLOYMENT_TARGET "10.9" CACHE STRING "Minimum OS X deployment version")
+project (huffman)
+
+set(CMAKE_C_STANDARD 99)
+set(CMAKE_C_STANDARD_REQUIRED ON)
+set(CMAKE_C_EXTENSIONS OFF)
+
+if(MSVC)
+    add_definitions (-D_WIN32_WINNT=0x0601)
+    add_definitions (-D_SCL_SECURE_NO_WARNINGS=1)
+    add_definitions (-D_CRT_SECURE_NO_WARNINGS=1)
+
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /EHsc /utf-8")
```

Comment:
I looked it up. /EHsc means standard C++ exception handling, and /utf-8 means treat the source files like UTF-8 (otherwise it would look for a BOM to determine).

Why do you need to specify C++ exception handling flags for this C only project?

Reply:
I am sorry for this mistake. I will fix it.